src/webapp/src/utils/utils.js

```diff
@@ -35,7 +35,7 @@ export default {
   docsUrl(path = '/', fragment) {
     fragment = fragment ? `#${fragment}` : ''
 
-    return `https://meltano.com/docs${path}.html${fragment}`
+    return `https://docs.meltano.com/${path}${fragment}`
```

Comment:
@alexmarple I'm not sure exactly what this affects, so I'm unsure where to look in the UI to test that the new links are correct. Also, I'm now noticing that there might be an extra slash before `${path}`.

Reply:
@WillDaSilva - Is it correct that this is just fixing links to docs? Do we know when this broke?

Reply:
@aaronsteers It just fixes the links to the docs, not where it's hosted. Here are the spots this change affects:
![image](https://user-images.githubusercontent.com/11428666/184947504-d65b98bc-efb3-4b51-b621-05789544e713.png)

Seems like it's limited to oauth. I wasn't aware there was any oauth in the web UI.

> Do we know when this broke?

These spots in the code haven't been updated in years, which suggests that it was just an oversight when we switched from hosting the docs at `meltano.com/docs` to `docs.meltano.com`.

Reply:
@aaronsteers @WillDaSilva This will just fix the url and not break anything.

There is some vestigial oauth stuff and i think this particular util is akin to that. There is a component that uses the util to provide a dynamic link but it's not actually used anywhere anymore.